### DISCUSSION TITLE
"valid" demo key expired

### DISCUSSION THREAD

**Discussion by nschloe** [keygen-sh/community#61] [Issue]
https://keygen.sh/demo/license-key-validation/

**Comment by ezekg** [comment]
Appreciate the heads up. Renewed.

**Reply by nschloe** [reply]
@ezekg Again.

**Reply by ezekg** [reply]
This is my yearly reminder to implement auto renewal: https://github.com/keygen-sh/keygen-api/issues/538 😂

Fixed.
